patches/net/minecraft/world/level/Explosion.java.patch

```diff
@@ -8,3 +8,12 @@
          Vec3 vec3 = new Vec3(this.x, this.y, this.z);
  
          for (Entity entity : list) {
+@@ -297,7 +_,7 @@
+                         d5 *= d10;
+                         d7 *= d10;
+                         d9 *= d10;
+-                        Vec3 vec31 = new Vec3(d5, d7, d9);
++                        Vec3 vec31 = net.neoforged.neoforge.event.EventHooks.onExplosionKnockback(this.level, this, entity, new Vec3(d5, d7, d9));
```

Comment:
Instead of having a diff that is a removal and an addition, this could just be a single line after `vec31`'s initialization:
```diff
Vec3 vec31 = new Vec3(d5, d7, d9);
+ vec31 = net.neoforged.neoforge.event.EventHooks.onExplosionKnockback(this.level, this, entity, vec31);
```